AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAssertImagePreviewActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertImagePreviewActionGroup">
+        <click selector="{{AdobeStockSection.imagesThumbnails}}" stepKey="clickOnThumbnail"/>
```

Comment:
This selector should be used together with "modal", please check how is it implemented here:
https://github.com/magento/adobe-stock-integration/blob/develop/AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAssertAdobeStockThumbnailsNumberActionGroup.xml

Please change it in the other action groups as well.

Reply:
ok